content/release-notes/1.19.0.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-09-14
+linktitle: "1.19.0"
+title: "1.19.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+  * Support for airgapped [headless installs](/kotsadm/installing/automating/).
```

Comment:
Is there a PR to update that page for airgapped? Or perhaps should it be included in this PR so it's merged at the same time?

Reply:
I think this is it: https://github.com/replicatedhq/kots.io/pull/271